tests/data_io/test_validate_dataframe.py

```diff
@@ -78,3 +78,17 @@ def test_failure_dict_return():
     result = dataframe.validate(df, raise_error=False, require_present_columns=["produce"])
     assert not result.passed
     assert "produce" in result.details["failed_present_columns"]
+
+
+def test_check_custom():
+    def check_for_value(value):
+        if value == 2:
+            return False
+        else:
+            return True
+
+    df_check = pd.DataFrame({"a": [1, 1, 1, 1], "b": [1, 1, 2, 2], "c": [2, 2, 2, 2]})
+    test_return = dataframe.validate(df_check, custom_check_columns=[check_for_value, ["a"]])
+    print(test_return)
+    with pytest.raises(ValueError):
+        dataframe.validate(df_check, custom_check_columns=[check_for_value, ["b", "c"]])
```

Comment:
Can you split this into two statements where you check that both "b" and "c" fail individually?